java/src/jmri/jmrit/symbolicprog/tabbedframe/PaneProgFrame.java

```diff
@@ -236,7 +242,22 @@ public boolean doPrompt() {
         // general GUI config
         pane.setLayout(new BorderLayout());
 
-        // configure GUI elements
+        // configure GUI buttons
+        ThreadingUtil.runOnGUIEventually( ()->{
+            configureButtons();
+        });
+        
+        // most of the GUI is done from XML in readConfig() function
+        // which configures the tabPane
+        pane.add(tabPane, BorderLayout.CENTER);
+
+        // and put that pane into the JFrame
+        getContentPane().add(pane);
```

Comment:
Just checking. `configureButtons()` is called on the GUI thread, but not the two `pane.add()` calls. Are these safe to call on a non GUI thread?